krakend/content.md

```diff
@@ -0,0 +1,144 @@
+%%LOGO%%
+
+# What is KrakenD?
+
+[KrakenD](https://www.krakend.io/) is a stateless, high-performance, enterprise-ready, open-source API gateway written in Go. Its engine (formerly known as *KrakenD Framework*) is now a **Linux Foundation Project** codenamed [Lura Project](https://luraproject.org/). Lura is the only enterprise-grade API Gateway hosted in a neutral, open forum.
+
+KrakenD is lightweight and straightforward as it only requires writing the configuration file. No Go knowledge is required. It offers connectivity to internal and external services, transformation and filtering of data, aggregation of multiple data sources (APIs, gRPC, queues and pub/sub, lambda, etc) simultaneously or in cascade, protects the access to your API, throughputs its usage, and integrates with a lot of third-parties.
+
+All features are designed to offer extraordinary performance and infinite scalability.
+
+## How to use this image
+KrakenD only needs a single configuration file to create an API Gateway, although you can have a complex setup reflecting your organization structure. The configuration file(s) can live anywhere in the container, but the default location is `/etc/krakend`.
+
+To use the image, `COPY` your `krakend.json` file inside the container or mount it using a volume. The configuration is checked only once during the startup and never used again. Don't have a config file yet? Generate it with the [KrakenD Designer UI](https://designer.krakend.io).
+
+⚠️ **NOTICE**: KrakenD does not use live reload when your configuration changes. Restart the container.
+
+### Quick start
+You can start an empty gateway with a health check with the following commands:
+
+```console
+$ echo '{ "version": 3 }' > krakend.json
```

Comment:
This is included in the image by default so is not required for a "quick start" so maybe should be covered in a separate block.

Reply:
Removed